core/mondoo-openssl-vulnerability.mql.yaml

```diff
@@ -1,6 +1,6 @@
 policies:
   - uid: mondoo-openssl-vulnerability
-    name: OpenSSL Vulnerability Policy by Mondoo
+    name: Mondoo OpenSSL Vulnerability Policy
```

Comment:
`by Mondoo` is how we've named all our other policies. We should change this one back or change all the other ones

Reply:
Thank you for the hint. I copied this from cnquery. I align it there.